index.ts

```diff
@@ -59,18 +89,18 @@ module.exports = function svelte(options = {}) {
 			return {code: '', moduleSideEffects: 'no-treeshake'};
 		},
 
-		generateBundle(options, bundle) {
-			if (pluginOptions.ignore!==false) return;
+		generateBundle(this: PluginContext, generateBundleOpts: NormalizedOutputOptions, bundle: OutputBundle) {
+			if (pluginOptions.ignore !== false || !generateBundleOpts.dir) return;
```

Comment:
We should ignore the processing and document it on the readme or warn the developer that he should provide an output directory?

Reply:
Yeah, providing a warning is a good idea. I've made that change